projects/plugins/jetpack/tests/php/modules/sso/test_class.jetpack-sso-helpers.php

```diff
@@ -1,407 +0,0 @@
-<?php
-require_once __DIR__ . '/../../../../modules/sso/class.jetpack-sso-helpers.php';
-
-/**
- * Testing functions in Jetpack_SSO_Helpers class.
- *
- * @since 4.1.0
- */
-class WP_Test_Jetpack_SSO_Helpers extends WP_UnitTestCase {
```

Comment:
Are these tests no longer needed, or we could reuse them for the new `Helpers` class?

Reply:
I moved them over to the new class in #36587.

They now live at https://github.com/Automattic/jetpack/blob/6438a55e74ce512ad33b10f34e5b92e9b368c010/projects/packages/connection/tests/php/sso/test_Helpers.php#L1-L15